### DISCUSSION TITLE
스크린샷 모듈을 가져와서 적용시 셀피카메라가 작동하지 않습니다.

### DISCUSSION THREAD

**Discussion by zepzino** [naverz/zepeto-studio-kor#720] [5. Q&A]
현재 카메라 모듈을 따로 빼어 작업중인데, 스크린샷 모듈그대로 가져와서 메타도 지우고 해봤는데도
셀피카메라로 진입시 작동이 되지 않고, 나올 경우에는 카메라가 고정되어 움직이지 않는 현상이 있습니다 ㅠ

아마 셀피카메라로 진입하면서 카메라 자체를 놓치는 것 같은데... 어떻게 수정해주어야 할까요?
도움 부탁드립니다 ㅠ

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, https://github.com/naverz/zepeto-studio-kor/discussions/437 가이드를 따라서 세팅을 확인해 보시고
그래도 안된다면 프로젝트를 공유해주시거나, 관련 코드 및 스크린샷을 첨부해주시기를 부탁 드립니다.
감사합니다.

**Reply by zepzino** [reply]
관련 스크린샷 및 파일 공유합니다~
[스크린샷관련공유.zip](https://github.com/naverz/zepeto-studio-kor/files/8656500/default.zip)

**Reply by zepzino** [reply]
증상 정확히 한번 더 말씀드리겠습니다.

1. 셀피 카메라 모드 진입시, 카메라가 캐릭터를 보지 않고 고정되는것으로 보이나 촬영시 셀피모드 사진이 촬영되어 나옴
2. 셀피모드 진입 후 취소하고 밖으로 나올시 카메라가 캐릭터를 비추지 않으며, 다시 촬영모드에서 셀피모드 -> 일반 모드로 변경하고 나오면 캐릭터가 보임
3. 상기 행위를 계속 반복하고나면 아예 먹통이 됨

**Reply by pumfit** [reply]
안녕하세요! 올려주신 파일 받아서 실행해보니 문제 없이 잘 실행되는 것 같습니다! 다만 올려주신 스크린샷에서 v1 으로 설정되어있는데 넣어준 애니메이터의 경우 v2로 보여서 해당 부분 버전이 맞게 들어가 있는지 확인해보시면 좋을 것 같습니다 또는 에러 로그가 있다면 올려주시면 더 도움이 될 것 같습니다!
